plotnine/geoms/geom.py

```diff
@@ -359,7 +359,9 @@ def _verify_arguments(self, kwargs):
         """
         Verify arguments passed to the geom
         """
-        unknown = (kwargs.keys() -
+        seen = set(kwargs.keys())
+        seen.update(self._stat._kwargs.keys())
+        unknown = (seen -
```

Comment:
Can you change it to

```python
        geom_stat_kwargs = kwargs.keys() ^ self._stat._kwargs.keys()
        unknown = (geom_stat_kwargs -
                   ...
```

Reply:
Not sure I follow - why would you want the symmetric difference of those two instead of their union? ( ^ vs | )

Reply:
Sorry, it should be It should be union `|`.